content/webapp/components/Contact/Contact.tsx

```diff
@@ -32,7 +32,7 @@ const PhoneNumber = styled.span.attrs({ className: font('intr', 4) })`
 `;
 
 const WithIconWrapper = styled(Space).attrs({
-  v: { size: 's', properties: ['margin-bottom', 'column-gap'] },
+  v: { size: 's', properties: ['margin-bottom', 'row-gap'] },
```

Comment:
orly? Was this a mistake before?

Reply:
yes and my fix was wrong too (changed it now). I think anything written in there _works_, but the _type_ isn't right. I added an `h` now for the `column-gap` as it was what we wanted, but shouldn't have been in the `v`.